pages.id/common/pip.md

```diff
@@ -12,9 +12,13 @@
 
 `pip install {{nama_paket}}=={{versi_paket}}`
 
+- Instal paket ke direktori pengguna alih-alih lokasi default seluruh sistem:
```

Comment:
@reinhart1010 could you please have a look at this? I think it was translated using an online service.

Reply:
```suggest
- Hanya memasang paket untuk digunakan oleh pengguna saat ini
```

Literally means `Only installs package for the current user`.

Reply:
Thanks!